plugins/osdn/ovs/bin/openshift-sdn-ovs

```diff
@@ -76,6 +79,30 @@ add_ovs_flows() {
 	ovs-ofctl -O OpenFlow13 add-flow br0 "table=7, priority=100, reg0=0, ip, nw_dst=${ipaddr}, actions=output:${ovs_port}"
 	ovs-ofctl -O OpenFlow13 add-flow br0 "table=7, priority=100, reg0=${tenant_id}, ip, nw_dst=${ipaddr}, actions=output:${ovs_port}"
     fi
+
+    # Pod ingress == OVS vSwitch egress
+    # linux-htb used here since that's the Kubernetes default traffic shaper too
+    if [ -n "${ingress_bw}" ]; then
+        qos=$(ovs-vsctl create qos type=linux-htb other-config:max-rate=${ingress_bw})
+        ovs-vsctl set port ${veth_host} qos=${qos}
```

Comment:
Yeah, that was left-over from some earlier experiments and I forgot the queues aren't actually needed unless you want to restrict traffic in a specific pattern.  Just setting a QoS on a specific interface with a max_rate is enough for this case.